themes/eds/index.js

```diff
@@ -98,4 +102,70 @@ module.exports = function (app) {
       dpId: JSON.stringify(datapackage).replace(/\\/g, '\\\\').replace(/\'/g, "\\'")//.replace(/'/g, "&#x27;")
     })
   })
+  app.get('/open-door-lab', async (req, res, next) => {
+    try {
```

Comment:
I don't think you need try catch statement here as there is no way this part can fail, e.g., no API call etc.